components/AirdropLogin.vue

```diff
@@ -109,21 +110,34 @@
 </template>
 <script lang="ts">
 import { Vue, Component, Prop } from 'vue-property-decorator'
-import { CONNECT_WALLET_TYPES } from '~/constant'
+import {
+  CONNECT_WALLET_TYPES,
+  COSMOS_ADDRESS_REGEX,
+  OSMO_ADDRESS_REGEX,
+} from '~/constant'
 
 @Component
 export default class AirdropLogin extends Vue {
   @Prop(Boolean) readonly isAirdropStarted: boolean | undefined
 
   inputAddress: string = ''
+  errorMessage: string = ''
 
   // eslint-disable-next-line class-methods-use-this
   get connectWalletTypes() {
     return CONNECT_WALLET_TYPES
   }
 
   handleAddressInput() {
+    this.errorMessage = ''
+    if (
+      !COSMOS_ADDRESS_REGEX.test(this.inputAddress) ||
+      !OSMO_ADDRESS_REGEX.test(this.inputAddress)
+    ) {
+      this.errorMessage = this.$t('AirDrop.errorMessage.address') as string
+    }
     this.$emit('input', this.inputAddress)
+    this.errorMessage = ''
```

Comment:
```suggestion
    this.errorMessage = ''
    if (
      !COSMOS_ADDRESS_REGEX.test(this.inputAddress) ||
      !OSMO_ADDRESS_REGEX.test(this.inputAddress)
    ) {
      this.errorMessage = this.$t('AirDrop.errorMessage.address') as string
      return;
    }
    this.$emit('input', this.inputAddress)
```

Reply:
`this.errorMessage` will never be shown in your version if you always empty it?

Reply:
soga